safety/scan/command.py

```diff
@@ -162,6 +167,12 @@ def process_report(
 
         if output is ScanOutput.JSON or ScanOutput.is_format(output, ScanOutput.SPDX):
             if output is ScanOutput.JSON:
+                if detailed_output and output is ScanOutput.JSON:
```

Comment:
nit: `output is ScanOutput.JSON` should be always true.

Reply:
ah removed it